packages/bpk-component-drawer/src/BpkDrawerContent.scss

```diff
@@ -70,7 +70,7 @@
 
   &__header {
     display: flex;
-    min-height: #{2 * $bpk-spacing-sm + $bpk-text-base-line-height}; // keep height the same if visually hidden title
+    min-height: #{2 * $bpk-spacing-sm + $bpk-spacing-base}; // keep height the same if visually hidden title
```

Comment:
Prep for when we remove line-height

Reply:
Won't this be too tall now? It's correct that `$bpk-text-base-line-height` and `$bpk-spacing-base` are both 1.5rem however the correct line height here is actually `$bpk-text-base-font-size * 1.3` which is `1.3rem`

Reply:
This is something that was pre existing and not aware of how that works but changing it from using the old line height to not keeps it the exact same as before all that changed is the naming

Reply:
I'm unsure if this is even needed. Quick test shows removing it has no effect regardless of title visibility. Need to be explored for other browsers though. Let's keep it for now